### ISSUE TITLE
how to predict for one sentence? 

### ISSUE THREAD

**Issue by prateeknepaliya09** [dennybritz/cnn-text-classification-tf#148]

**Comment by sonu107** [comment]
Delt all your test data. And just keep one single line.

**Comment by Nadedic** [comment]
I've trained till I've reached an accuracy of 1, almost stable. How can I evaluate or test it? How will run eval file exactly? Also isn't testing on the same training data might cause overfit?

**Comment by wangtao-humanitarian** [comment]
same question

**Comment by bibhu107** [comment]
@Nadedic Yes! We don't test with same data. That wont produce the exact accuracy of the model. If you have trained then you will be having  a text file having word->ID use it and make an embedding layer then  see eval.py .
